woocommerce/wc-api-dev
# Fix for class issues in 3.6 and version bump.

While attempting to test the beta of 3.6 on a pressable site ( which was created using WPCOM + Store, and as such this plugin is loaded ) - I was getting the following fatal:

> 2019-03-20T18:46:20+00:00 CRITICAL Class 'WC_REST_Coupons_Controller' not found in /wordpress/plugins/wpcomsh/2.4.5/vendor/woocommerce/wc-api-dev/api/class-wc-rest-dev-coupons-controller.php on line 22

It seems the base Woo API classes were yet to be loaded, and as such I moved the loading of the endpoint files into the `rest_api_init` hook instead.

__To Test__
- Install WooCommerce + beta tester plugin
- Install this plugin, verify things work with 3.5.x
- Update to 3.6 beta1, and verify no errors are thrown

I have bumped versions in this PR too, to save a step when prepping a release.
